torchvision/datasets/utils.py

```diff
@@ -1,7 +1,11 @@
 import os
 import os.path
 import hashlib
+import gzip
 import errno
+import tarfile
+import zipfile
```

Comment:
Shouldn't we do a lazy import in `extract_file()`? AFAIK we do this now to prevent becoming dependent on these packages at `import`.

Reply:
I believe `tarfile`, `zipfile` and `gzip` are part of the python standard library, so I think this should be ok